__mocks__/schemas/ComprehensiveSourceSchema.json

```diff
@@ -0,0 +1,454 @@
+{
+  "name": "ComprehensiveSource.xsd",
+  "type": "XML",
+  "targetNamespace": "http://tempuri.org/ComprehensiveSource.xsd",
+  "namespaces": {
+    "ns0": "http://tempuri.org/ComprehensiveSource.xsd",
+    "xs": "http://www.w3.org/2001/XMLSchema"
+  },
+  "schemaTreeRoot": {
+    "key": "/ns0:SchemaRoot",
+    "name": "SchemaRoot",
```

Comment:
I might update this to "Source" (SourceSchemaRoot) and all the subsequent ones, that way it's easy to look at a map and see that source/target is being assigned to the correct spots, even if the names are the same